content/community/webinars.md

```diff
@@ -8,12 +8,17 @@ g3Teaser:
   title: Gen3 Webinars
   detail: The Gen3 webinars are focused on helping the community understand what Gen3 does, what the Data Commons Framework Services (DCFS) are, and how both can be used to aid in the construction and operation of data commons.
 g3Upcoming:
-  - Title: Gen3 Data Modeling - Herding Data Submissions & Hunting Down Data (Sheepdog & Peregrine)
-    Date: May 9th, 2019 at 1 PM U.S. Central Time.
-    Description: This webinar will cover the creation of data models, and data dictionaries in Gen3. 
+  - Title: Data Science with Gen3 - Using Jupyter Notebooks
+    Date: June 13th, 2019 at 1 PM U.S. Central Time.
+    Description: Join us as we walk through Gen3’s services and demonstrate how researchers can bring data into Gen3 and get access to a data ecosystem with open-source tools to drive scientific discovery. During this webinar we will dive into the details of data science with Gen3 and how Jupyter Notebooks work with Gen3.
     Image: coming_soon.svg
-    Link: https://www.eventbrite.com/e/gen3-data-modeling-herding-data-submissions-hunting-down-datasheepdog-peregrine-tickets-58469774670?aff=gen3
+    Link: https://www.eventbrite.com/e/data-science-with-gen3-using-jupyter-notebooks-tickets-58824297056?aff=gen3
 g3Previous:
+  - Title: Gen3 Data Modeling - Herding Data Submissions & Hunting Down Data (Sheepdog & Peregrine)
+    Date: May 9th, 2019
+    Description: This webinar will cover the creation of data models, and data dictionaries in Gen3.
+    Youtube: cVTvzP-li0M
+    Slides: Webinar_20190509.pdf
```

Comment:
Is this a link or what?

Reply:
Yes, this gets transformed in the template to a youtube link & youtube preview. Check http://alpha.gen3.org/community/webinars/ to see it